be/src/storage/primary_index.h

```diff
@@ -161,6 +161,8 @@ class PrimaryIndex {
 
 protected:
     void _set_schema(const Schema& pk_schema);
+    const Slice* _build_persistent_keys(const Column& pks, uint32_t idx_begin, uint32_t idx_end,
```

Comment:
I'm a little confused that this function returns a `Slice*`. Because a `Slice` consists of a pointer and a length, which is not a big memory footprint, is it necessary to return a `Slice*`

Reply:
You can take a look about `_build_persistent_keys`, it return the start pointer of slice array.

Reply:
Add comments.